app/assets/stylesheets/govuk-component/_taxonomy-sidebar.scss

```diff
@@ -1,29 +1,48 @@
 .govuk-taxonomy-sidebar {
-  border-top: 10px solid $mainstream-brand;
-  padding-top: 5px;
+  border-top: 2px solid $govuk-blue;
+  padding-top: $gutter-half;
   @include core-16;
 
-  h2 {
-    @include bold-24;
-    margin-top: 0.3em;
-    margin-bottom: 0.5em;
+  .govuk-taxonomy-sidebar__heading {
+    margin-top: 0;
+    margin-bottom: $gutter-one-third;
+    @include bold-19;
   }
 
   .taxon-description {
-    margin-bottom: 0.75em;
+    margin-top: 0;
+    margin-bottom: $gutter-one-third;
+  }
+
+  a {
```

Comment:
We usually wouldn't style an "a" directly. Sometimes we are forced to, but we absolutely need to ensure we're not styling all a tags on the page, so 
.taxon-description a { blah } 
or something may be necessary - ideally though the anchor would have a styling class such as .govuk-taxonomy-sidebar__link

Reply:
These seem to be nested within the selector at the top, but massive 👍 to always avoiding styling base elements if possible.

We should only be doing this if there's no other way as it directly conflicts with our naming conventions that allow us to have confidence with components across so many applications.

https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md

Reply:
Specifically we're using [BEM](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md#bem) and [namespacing](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md#namespacing)